packages/react-sketch-canvas/src/Canvas/index.tsx

```diff
@@ -291,13 +311,31 @@ release drawing even when point goes out of canvas */
     );
   }, [paths]);
 
+  // avoid pen from scrolling if scrollOnTouch
+  useEffect(() => {
+    const listener = function(
```

Comment:
It's better to move this listener as a callback using `useCallback` with empty dependency array.